### DISCUSSION TITLE
Where's the best place to get help for a newb?[Support]:

### DISCUSSION THREAD

**Discussion by yourpalfranc** [blakeblackshear/frigate#13858] [General]
I'm a complete newb to Frigate trying to set up a test machine and hoping to replace Blue Iris someday. But, what a learning curve???? I had a working version in Docker on an Intel i7 PC and also have Portainer installed, but I tried to restart the container after a config.yaml change, and I can no longer display the webUI on port 8971. Where's the best place to ask for a helping hand? Here?? I'm very open to learning, have the time, and will very much appreciate the help.

**Comment by NickM-27** [comment]
yes, this is the right place. 

For starters, all config changes should really be done inside of the webUI config editor as it will stop you from saving an invalid config and also has the option to restart itself. You will want to create a support discussion

**Reply by yourpalfranc** [reply]
OK. Great. I didn't see that mentioned in any of the getting started docs.

I'll create a discussion in the General Support on my current state. Thanks!
